src/layers/viv-viewer-layer/viv-viewer-layer.js

```diff
@@ -65,6 +73,39 @@ export class MicroscopyViewerLayer extends CompositeLayer {
     }
   }
 
+  _overrideChannelProps() {
+    const { sliderValues, colorValues, channelsOn } = this.props;
+    const orderedChannelNames = Object.keys(sliderValues);
```

Comment:
Do you need a sort? Or safe to assume object keys are ordered in ES?

Reply:
We're using the order given - this is a deliberate choice.